uxarray/_scrip.py

```diff
@@ -112,3 +113,87 @@ def _read_scrip(ext_ds):
             "for more information on SCRIP Grid file formatting")
 
     return ds
+
+
+def _write_scrip(ext_ds, outfile):
+    """Function to reassign UGRID formatted variables to SCRIP formatted
+    variables.
+
+    Currently, supports creating unstructured SCRIP grid files following traditional
+    SCRIP naming practices (grid_corner_lat, grid_center_lat, etc).
+
+    Unstructured grid SCRIP files will have 'grid_rank=1' and include variables
+    "grid_imask" and "grid_area" in the dataset.
+
+    More information on structured vs unstructured SCRIP files can be found here:
+    https://earthsystemmodeling.org/docs/release/ESMF_6_2_0/ESMF_refdoc/node3.html
+
+    Parameters
+    ----------
+    ext_ds : :class:`xarray.Dataset`
+        Original UGRID dataset of interest being used
+
+    outfile : :class:`string`
```

Comment:
You might considering architecting this as an "encoder" function.

That way you can

```
ds = grid.encode_scrip()  # or various other formats
# Now ds.to_netcdf or ds.to_zarr or various other formats
```

Reply:
Good suggestion! I've created Issue #109 to address this. This PR could just be merged (if it is ready), and that issue could be addressed afterwards as a refactoring effort. 

@rajeeja @anissa111 @dcherian  thoughts?